code/wdb/datastripe.q

```diff
@@ -2,26 +2,57 @@
 
 segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
 
-td:hsym `$getenv`KDBTAIL
+td:hsym `$"/"sv (getenv`KDBTAIL;string .z.d)
```

Comment:
I've added .z.d to the directory path so that when I am loading the access table from disk I only have the access table for one day. I believe changes will need to be made to the end of day function to move and clear the access table on disk. I will make a note of this. td is the tail directory the tables are saved to interday. I will add a comment to this.